en/config/safety.md

```diff
@@ -62,9 +62,10 @@ Battery Warn Level | [BAT_LOW_THR](../advanced_config/parameter_reference.md#BAT
 Battery Emergency Level | [BAT_EMERGEN_THR](../advanced_config/parameter_reference.md#BAT_EMERGEN_THR) | Percentage capacity for triggering Land (immediately) action.
 
 
-### RC Loss Failsafe
+### Manual Control Loss failsafe
```

Comment:
I did not know this!

Reply:
Yes I think we should - fyi @MaEtUgR maybe you could take care of that?

Reply:
I took a shot at it in https://github.com/PX4/PX4-Autopilot/pull/22150